commands/new_function.go

```diff
@@ -114,6 +115,14 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("got unexpected error while updating .gitignore file: %s", err)
 	}
 
+	var imageName string
+	imagePrefix = strings.TrimSpace(imagePrefix)
```

Comment:
added trimming of the prefix parameter

Reply:
This looks fine. It is also OK to in-line it but I won't get picky.